app/assets/stylesheets/projects.scss

```diff
@@ -1,85 +1,99 @@
+.projects-background { 
+  width: 100%;
+  background-color: #eaeaea;
+}
+
+.projects-section {
+  @extend .columns;
+  @extend .small-11;
+  @extend .small-centered;
+}
 
-  .project-block {
+.project {
+  padding: 48px 0px 0px;
+  @extend .row;
+
+  .projects-image {
     @extend .columns;
     @extend .small-12;
-    @extend .medium-6;
-    @extend .large-4;
+    @extend .medium-4;
+    @extend .large-5;
     height: $block-height;
-    border: 10px solid rgba(255, 255, 255, 1);
-    padding:0px;
-    background-color:$blue;
-    background-size:cover;
-    display:inline-block;
-    position:relative;
-    box-shadow:0px -45px 85px 0px rgba(0, 0, 0, 0.4) inset;
-    cursor:pointer;
-    transition-duration:0.3s;
-    overflow:hidden;
-
-    .project-title {
-      position:absolute;
-      bottom:0px;
-      padding:5px 15px;
-      width:100%;
-      transition-duration:0.3s;
-      h3 {
-        color: white;
-        text-align:right;
-      }
-      a {
-        text-decoration: none;
-      }
-    }
+    background-size: cover;
+    box-shadow: 3px 3px 2px #CCCCCC;
+    border-radius: 2px 0px 0px 2px;
+  }
 
-    .project-text {
-      position:absolute;
-      top:100%;
-      height:100%;
-      width:100%;
-      display:block;
-      overflow:hidden;
-      background-color:rgba(0, 0, 0, 0.7);
-      transition-duration:0.3s;
-      color:#fff;
-      padding: 5px;
+  .projects-card {
+    @extend .columns;
+    @extend .small-12;
+    @extend .medium-8;
+    @extend .large-7;
+    height: $block-height;
+    background-color: white;
+    box-shadow: 3px 3px 2px #CCCCCC;
+    border-radius: 0px 2px 2px 0px;
 
+    .projects-card-text {
+      color: #565a5c;
+      text-align: left;
+    }
+    .projects-title {
+      @extend .projects-card-text;
+      padding: 0px 8px;
+      height: 57px;
+      line-height: 57px;
+      overflow: hidden;
+    }
+    .projects-description {
+      @extend .projects-card-text;
+      font-size: 16px;
+      line-height: 16px;
+      padding: 0px 8px;
+      height: 48px;
+    }
+    .projects-subtitle {
+      @extend.projects-card-text;
+      font-size: 14px;
+      line-height:14px;
+      padding: 12px 8px 12px;
+      display: block;
+    }
+    .project-member-images {
+      margin-left: 4px;
+    }
+    .project-member-image {
+      display: inline-block;
+      height: 48px;
+      width: 48px;
+      border-radius: 24px;
+      background-color: $blue;
+      margin: 4px 4px;
+    }
+    .projects-progress {
+      margin-left: 8px;
     }
 
-    &:hover {
-      box-shadow:0px -50px 95px 0px rgba(0, 0, 0, 0.5) inset;
-      .project-title {
-        bottom:$block-height;
-      }
-      .project-text {
-        top:0px;
-      }
+    .projects-progress-scope {
+      display: inline-block;
+      height: 8px;
+      width: 20%;
+      margin: 0px 2px 4px 0px;
+      background-color: #FF7E79;
     }
   }
 }
+.projects-banner {
+  background-image: url("projects/projects-banner.png");
```

Comment:
You should use `image-url` to make sure it works on Heroku. http://guides.rubyonrails.org/asset_pipeline.html#css-and-sass